src/components/navbar.js

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import {BrowserRouter as Router, Switch, Route, Link} from "react-router-dom";
+
+class Navbar extends React.Component{
```

Comment:
Please refactor in a functional component for now.

Reply:
@bogdandan done! You can see the changes below